replicator/src/main/scala/com/evolutiongaming/kafka/journal/replicator/ReplicateRecords.scala

```diff
@@ -72,11 +72,13 @@ private[journal] object ReplicateRecords {
           def measure(events: Nel[EventRecord[EventualPayloadAndType]], expireAfter: Option[ExpireAfter]) = {
             for {
               measurements <- measurements(records.size)
-              version       = events.last.version.map(_.value).getOrElse("none")
+              version       = events.head.version.map(_.value).getOrElse("none")
```

Comment:
Please elaborate why `events.last` was changed on `events.head`?

Reply:
for `version` it plays no role, but for `expiration` it might in scenarios when journal gets several different values depending on state.

my idea was that with this approach we would get wider (and actual) list of values, though as this happens on per-poll and Kafka might batch in different ways, it should not make much difference in long-term